### PR TITLE
IA-2405: Rename frontend permissions

### PR DESCRIPTION
**Pull request by hakifran**
Explain what problem this PR is resolving
- Change the permissions names in translation

Related JIRA tickets : [IA-2405](https://bluesquare.atlassian.net/browse/IA-2405)

## Self proofreading checklist

- [x] Did I use eslint and black formatters
- [x] Is my code clear enough and well documented
- [ ] Are my typescript files well typed
- [x] New translations have been added or updated if new strings have been introduced in the frontend
- [ ] My migrations file are included
- [ ] Are there enough tests
- [ ] Documentation has been included (for new feature)

## Changes
- Change the translations of permissions in french and english
## How to test
- Try to edit or create a user or a user role and check in permissions tab

## Print screen / video

# Before
Screenshot from 2023-09-21 14-27-08
# After
Screenshot from 2023-09-21 14-27-23

[IA-2405]: https://bluesquare.atlassian.net/browse/IA-2405?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by hakifran** [comment]
@cypress

File: hat/assets/js/apps/Iaso/domains/app/translations/en.json [hunk 1]
```diff
@@ -793,42 +793,42 @@
...
     "iaso.periods.end": "End period",
     "iaso.periods.start": "Start period",
-    "iaso.permission.reports": "Reports",
-    "iaso.permissions.assignments": "Assignments",
-    "iaso.permissions.completeness": "Data completeness",
-    "iaso.permissions.completeness_stats": "Completeness stats",
-    "iaso.permissions.dataTasks": "Tasks",
-    "iaso.permissions.entities": "Beneficiaries",
-    "iaso.permissions.forms": "Forms",
+    "iaso.permission.reports": "Devices",
```
Comment (quang-le) [inline hat/assets/js/apps/Iaso/domains/app/translations/en.json:796]: "Reports" is "devices" ???

File: hat/assets/js/apps/Iaso/domains/app/translations/en.json [hunk 2]
```diff
@@ -793,42 +793,42 @@
...
+    "iaso.permissions.forms": "Forms management",
     "iaso.permissions.iaso_dhis2_link": "Link with DHIS2",
-    "iaso.permissions.iaso_entity_duplicates_read": "See entity duplicates",
-    "iaso.permissions.iaso_entity_duplicates_write": "Edit entity duplicates",
-    "iaso.permissions.iaso_entity_type_write": "Beneficiary type create/update/delete",
-    "iaso.permissions.iaso_org_unit_groups": "Org unit groups",
-    "iaso.permissions.iaso_org_unit_types": "Org unit types",
-    "iaso.permissions.iaso_page_write": "Edit page",
+    "iaso.permissions.iaso_entity_duplicates_read": "Entity duplicates - Read only",
+    "iaso.permissions.iaso_entity_duplicates_write": "Entity duplicates - Read and Write",
```
Comment (quang-le) [inline hat/assets/js/apps/Iaso/domains/app/translations/en.json:805]: Are we sure that the write permission always gives access to the functionalities of the read permission?

**Review by quang-le — COMMENTED** [review]
Left some comments. Don't know how relevant they are though

File: hat/assets/js/apps/Iaso/domains/app/translations/en.json (hunk 1 above)
Comment (hakifran) [reply hat/assets/js/apps/Iaso/domains/app/translations/en.json:796]: @quang-le 
Screenshot from 2023-09-22 15-45-12

File: hat/assets/js/apps/Iaso/domains/app/translations/en.json (hunk 2 above)
Comment (hakifran) [reply hat/assets/js/apps/Iaso/domains/app/translations/en.json:805]: @quang-le it  is from the ticket Laure created.
Screenshot from 2023-09-22 15-51-17

**Comment by hakifran** [comment]
> Left some comments. Don't know how relevant they are though

@quang-le I asked the questions you asked to Laure about the replacement of reports with devices and she confirmed that it's okay, reports is renamed into devices

**Review by beygorghor — APPROVED** [review]

**Pull request merged**